app/components/assets/assets-index/filters.tsx

```diff
@@ -124,14 +123,17 @@ export function AssetIndexFilters({
               }}
               renderItem={({ metadata }) => (
                 <div className="flex items-center gap-2">
-                  <Image
-                    imageId={metadata.imageId}
-                    alt="img"
-                    className={tw(
-                      "size-6 rounded-[2px] object-cover",
-                      metadata.description ? "rounded-b-none border-b-0" : ""
-                    )}
-                  />
+                  {metadata?.thumbnailUrl ? (
```

Comment:
Okey so here we are going a bit out of the original scope, but because we are working on the images already, i want to do this anyways. 2 changes need to be done:
1. I want to implement the same component as with assets, namely a `LocationImage` component that works also with a preview and thumbnail.
2. Refactor Location page:
- [ ] Place the image next to the tittle - same as with assets and kits
- [ ] Remove the image from the right column
- [ ] Removing the image from the right column will make it look weird, so can you create an empty state for the address so there is always something there?